homeassistant/components/shelly/event.py

```diff
@@ -95,6 +103,21 @@ async def async_setup_entry(
                 async_remove_shelly_entity(hass, EVENT_DOMAIN, unique_id)
             else:
                 entities.append(ShellyRpcEvent(coordinator, key, RPC_EVENT))
+
+        script_instances = get_rpc_key_instances(
+            coordinator.device.status, SCRIPT_EVENT.key
+        )
+        for script in script_instances:
```

Comment:
Our [BLE script](https://github.com/home-assistant-libs/aioshelly/blob/61620d7e18861cae86cb1c045915c9b317ce29cd/aioshelly/ble/const.py#L33) uses `Shelly.emitEvent` method and generates a lot of events that will not be useful to the user. I wonder if we should skip this script.